### DISCUSSION TITLE
Connecting App away from home

### DISCUSSION THREAD

**Discussion by callrescue** [chuot/rdio-scanner#337] [Q&A]
I am trying to self-host and have a domain through Cloudflare. I can only connect through localhost:3000. I have it connected, and I am trying to set it up so that I can enter the website in the url and not the ip address and access away from home.  I have tried everything I can find, but still am unsuccessful. I believe one my issues is that I'm not sure if my DNS is pointing my domain. When I go to my domain, I get the following error: "The initial connection between Cloudflare's network and the origin web server timed out. As a result, the web page can not be displayed." I am using Windows 11 and SDRTrunk. 
Any help would be greatly appreciated!

**Comment by wmg819** [comment]
Have you done the steps shown on page 5 of the rdio-scanner pdf file that's with the program showing how to set up rdio-scanner to listen on ports 80 and 443? Following those steps and setting up the start batch file are needed to run the rdio-scanner connecting from away from home using  just a domain name. Also you may need to list rdio-scanner in your routers firewall to allow access to those ports. Then make sure your domain A record on cloudflare is pointing to your home ip address where rdio-scanner is running.  Hopefully this makes sense and helps, it's what I did https://www.rowannc911scanner.com

**Reply by Thinline-Dynamic-Solutions** [reply]
So your aware it says you have an invalid SSL certificate.

**Reply by wmg819** [reply]
No ssl cert wasn't expiring until June 2023 so I need to look in to that.

**Comment by Thinline-Dynamic-Solutions** [comment]
> Replies

The pdf file is correct when you follow the steps. I personally use noip, works great without any issues. Check out my website www.radioscannernetwork.com and also my scanner feed. https://ohiorsn.ddns.net  Access Code Until 3/20/2023: 24 
normally run access code is: demo
Doubble check your windows firewall settings as well if that is the platform you are running on.
